### DISCUSSION TITLE
阿里云盘open获取不到token

### DISCUSSION THREAD

**Discussion by gwx-7** [AlistGo/alist#4373] [General]
阿里云盘open获取刷新令牌失败

**Comment by m1570908333** [comment]
> 阿里云盘open获取刷新令牌失败

哥们你是不是显示这个错误：failed get enabled storages: AccessTokenExpired:token expired

**Reply by gwx-7** [reply]
对 我把代理关了就好了

**Reply by m1570908333** [reply]
> 对 我把代理关了就好了

想请教一下，代理哪里关呀？

**Reply by gwx-7** [reply]
要看你用的啥插件 反正就是把科学上网关掉 直接裸连。我是这样的解决的

**Reply by m1570908333** [reply]
我把代理关了之后，就出现这样了：failed get objs: failed to list objs: AccessTokenExpired:JWT is expired at `2023-05-19T04:07:30Z`

**Reply by m1570908333** [reply]
你有出现过这个问题吗

**Reply by gwx-7** [reply]
这我就不知道了 没有出现过这种情况
